### ISSUE TITLE
Handle name collisions gracefully

### ISSUE THREAD

**Issue by kfcampbell** [DanceParty/MSHackGeoGuesser#28]
Currently if you try to set your initials to initials that have already been used (and you're not logged in as that person anymore), PlayFab will return a 400 and log it, but we won't show any indication to the user.

**Comment by DanceParty** [comment]
I cannot recreate this :(

**Comment by kfcampbell** [comment]
Hmm...I don't seem to be able to repro now either. If I can't do it soon, I'll go ahead and close this issue.

**Comment by DanceParty** [comment]
```
PlayFabClientApi.js:130 Uncaught (in promise) {code: 400, status: "BadRequest", error: "NameNotAvailable", errorCode: 1058, errorMessage: "Name not available", …}
```

**Comment by DanceParty** [comment]
Should be fixed in #38
